### PR TITLE
Fix install warning layout

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-frontend/issues/8540

Before:

Screenshot 2019-09-03 10 40 26

Screenshot 2019-09-03 11 51 07

After:

Screenshot 2019-09-03 11 00 16
Screenshot 2019-09-03 11 51 20

### CODE REVIEW

File: src/amo/pages/Addon/index.js [hunk 1]
```diff
@@ -477,7 +477,12 @@ export class AddonBase extends React.Component {
                 {i18n.gettext('Extension Metadata')}
               </h2>
             </header>
-            {addon && <InstallWarning addon={addon} />}
+            {addon && (
+              <InstallWarning
+                addon={addon}
+                className="Addon-install-warning-notice"
```
Comment (bobsilverberg) [inline src/amo/pages/Addon/index.js]: I believe that we can use `width: 100%` for `InstallWarning` everywhere, so we don't really need to allow for a `className`for `InstallWarning`. We can just change its default style.

File: src/amo/components/InstallWarning/index.js [hunk 2]
```diff
@@ -122,15 +124,15 @@ export class InstallWarningBase extends React.Component<InternalProps> {
...
+    const { className, i18n, variant } = this.props;
 
     if (this.couldShowWarning() && variant === VARIANT_INCLUDE_WARNING) {
       return (
         <Notice
           actionHref={WARNING_LINK_DESTINATION}
           actionTarget="_blank"
           actionText={i18n.gettext('Learn more')}
-          className="InstallWarning"
+          className={makeClassName('InstallWarning', className)}
```
Comment (bobsilverberg) [inline src/amo/components/InstallWarning/index.js]: We don't really need this anymore.

**Comment by kumar303** [comment]
For anyone following the thread here, we discussed the centering issue on IRC. I suggested that we continue to follow the [Photon spec](https://design.firefox.com/photon/components/message-bars.html) which calls for left-aligned content. Putting the call-to-action button on the right might strike more balance but I think there are accessibility benefits to going with the left-alignment, especially when a fluid screen can become very wide.

**Comment by kumar303** [comment]
Thanks for the quick reviews!

**Comment by bobsilverberg** [comment]
Thanks for the quick fix!

**Pull request merged**